csunplugged/tests/resources/views/test_binary_to_alphabet.py

```diff
@@ -0,0 +1,154 @@
+from django.urls import reverse
+from tests.BaseTestWithDB import BaseTestWithDB
+from tests.resources.ResourcesTestDataGenerator import ResourcesTestDataGenerator
+from utils.create_query_string import query_string
+
+
+class BinaryToAlphabetResourceViewTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.test_data = ResourcesTestDataGenerator()
+        self.language = "en"
+
+    def test_binary_to_alphabet_resource_form_view(self):
+        resource = self.test_data.create_test_resource(
+            "binary-to-alphabet",
+            "Binary To Alphabet",
+            "resources/binary-to-alphabet.html",
+            "binary_to_alphabet.py"
+        )
+        kwargs = {
+            "resource_slug": resource.slug
+        }
+        url = reverse("resources:resource", kwargs=kwargs)
+        response = self.client.get(url)
+        self.assertEqual(200, response.status_code)
+
+    def test_binary_to_alphabet_resource_generation_student_a4(self):
```

Comment:
Sorry, must have missed the notification on this one (wish GitHub was more specific about the type of notification).

See this function from `test_binary_cards_small`: 
```
def test_binary_cards_small_resource_generation_valid_configurations(self):
        resource = self.test_data.create_resource(
            "binary-cards",
            "Binary Cards (small)",
            "resources/binary-cards-small.html",
            "binary_cards_small.py",
        )
        kwargs = {
            "resource_slug": resource.slug,
        }
        base_url = reverse("resources:generate", kwargs=kwargs)
        resource_module = import_resource_module(resource)
        valid_options = resource_module.valid_options()
        valid_options["header_text"] = ["", "Example header"]
        valid_option_keys = sorted(valid_options)
        combinations = [dict(zip(valid_option_keys, product)) for product in itertools.product(*(valid_options[valid_option_key] for valid_option_key in valid_option_keys))]  # noqa: E501
        print()
        for combination in combinations:
            print("   - Testing combination: {} ... ".format(combination), end="")
            url = base_url + query_string(combination)
            response = self.client.get(url)
            self.assertEqual(200, response.status_code)
            if combination["dot_counts"] == "yes":
                display_numbers_text = "with dot counts"
            else:
                display_numbers_text = "without dot counts"
            if combination["black_back"] == "yes":
                black_back_text = "with black back"
            else:
                black_back_text = "without black back"
            subtitle = "{} bits - {} - {} - {}".format(
                combination["number_bits"],
                display_numbers_text,
                black_back_text,
                combination["paper_size"],
            )
            self.assertEqual(
                response.get("Content-Disposition"),
                'attachment; filename="Resource Binary Cards (small) ({subtitle}).pdf"'.format(subtitle=subtitle)
            )
            print("ok")
```